src/appkit.cs

```diff
@@ -22250,6 +22708,11 @@ partial interface NSText {
 
 		[Notification, Field ("NSTextDidChangeNotification")]
 		NSString DidChangeNotification { get; }
+
+		//radar 32929318 - Does not exist in binaries
+		//[Mac (10, 13)]
+		//[Notification, Field ("NSTextMovementUserInfoKey")]
```

Comment:
WARNING: This is not a `[Notification]` but a key for a notification payload. Please remove the `NotificationAttribute` from the commented code so we do not forget.